.github/workflows/publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish
```

Comment:
Feels like this might have been accidentally included in this PR rather than deliberately, I sense I rebase in your future :D

Reply:
Yep. That's exactly it!